simulation/src/main/kotlin/it/unibo/collektive/examples/chat/ChatConstants.kt

```diff
@@ -0,0 +1,10 @@
+package it.unibo.collektive.examples.chat
+
+/**
+ * Maximum distance at which the message is clearly received.
+ */
+public const val REACHABLE: Double = 5.0
+/**
+ * Maximum distance at which the message can still be faintly perceived.
+ */
+public const val THRESHOLD: Double = 10.0
```

Comment:
```suggestion
const val REACHABLE: Double = 5.0
/**
 * Maximum distance at which the message can still be faintly perceived.
 */
const val THRESHOLD: Double = 10.0
```
There's no need to annotate them as `public`, since it is the default visibility for kotlin